developer/src/kmc/src/util/baseOptions.ts

```diff
@@ -23,6 +23,10 @@ export class BaseOptions {
     return program.addOption(new Option('-l, --log-level <logLevel>', 'Log level').choices(ALL_COMPILER_LOG_LEVELS).default('info'));
   }
 
+  public static addLogFormat(program: Command) {
+    return program.addOption(new Option('-l, --log-format <logFormat>', 'Log format').choices(ALL_COMPILER_LOG_FORMATS).default('formatted'));
```

Comment:
Does `-l` need to be unique for `--log-level`? (l. 23)

Reply:
Good catch!

```suggestion
    return program.addOption(new Option('--log-format <logFormat>', 'Log format').choices(ALL_COMPILER_LOG_FORMATS).default('formatted'));
```